src/software/fastSine.cpp

```diff
@@ -52,4 +52,13 @@ const int16_t sineTable[SINE_VAL_COUNT] = {
 In 1/32 microstep mode,  every sine wave index point is used. (0, 1, 2, ..., 127, 128==0, ...)
 In full step mode, only 4 index points of  sine wave are used. (0, 1/2 pi, pi, 3/2pi, 2pi, ...) == (0, 32, 64, 96, 128==0, ...)
 The rest of the modes are in between them.
+
+Microstep  |Frequency of input | Shaft, | Number of input | Number of shaft
+divider    | pulses, Hz        |   RPM  | pulses          | rotations
+      1    |        200        |    60  |     200 * N     |     N
```

Comment:
Are you planning on adding these?

Reply:
>What is the N for?

The N is a natural number, 1,2,3,4,5,......

>Are you planning on adding shaft rotations? 

No, There is encoder RPM in OLED.

>Should I remove it? 

Please save this for history.

Reply:
Thanks, that clears it up. I didn't recognize the n as a variable.